### ISSUE TITLE
Review accessibility of theme so far for site in "Peri" state

### ISSUE THREAD

**Issue by jscaffrey** [code4lib/2021.code4lib.org#157]
How to view this state without it being live:

1. Make sure you have the most up to date version: `git fetch` then `git pull origin main`
2. In _data/conf.yml set `homepage-display: peri`
3. Serve the site out as usual like `bundle exec jekyll serve`

You can either make new issues for each finding or list them out here.

Head's up: I suspect the skip-link may be broken on the Schedule page but didn't investigate in depth.

**Comment by yarnetsky** [comment]
I'll leave comments here to give y'all the power to decide how the possible errors are parsed into issues. Holler if you have any questions :) (I'll dig into the inside pagers in a moment)

## Created issues....

~~The skip-to-content link is indeed acting oddly. When clicked, it seems to jump to the right part of the page, but then the focus jumps back to the skip to main link. (tested with VoiceOver). I can't say I've seen that before, but if you want a second set of eyes for possible solutions, just holler.~~
~~The :focus indicator nearly disappears on the register and sponsor buttons.~~
~~The event cards should not have repeating links. The best option I know of would be to include an attribute of `tabindex="-1"` to the sun/moon svg link elements. The icons will still be clickable, but when person using a screen reader tabs through the cards, each will announces that day just once instead of twice.~~
~~The text in the event cards (ie Awesome talks, Keynote) are very low contrast — 2.2:1 currently. We need to get to 4.5:1~~

### remaining low priority issues
- Link names and titles don't match. The aria-label correctly announces the link name so a screen reader won't have an issue, but a person who looks at the link urls might be slightly confused.
- The testimonials section doesn't appear to have any aria-oriented labeling. A header or the section opening tag could include attribute of `aria-label="testimonials"` would do the trick.

**Comment by yarnetsky** [comment]
## Inside page observations/issues

~~Individual Day schedule pages lacking id="main" for the skip to main link.~~
~~Speakers, Schedule, and other pages that do have #main have same bounce back to skip link error as homepage.~~
~~On speakers page, the link inside the speaker-info box captures the tab whether the speaker is active or not. Needs JS to set attribute for link to `tabindex="-1"` unless the speaker box is .selected, then flip to `tabindex="0"`~~

## Other observations while testing accessibility

- The arrows pointing to the geometric globes strike me as signifiers for an action, but there is none. Maybe keep the globe but lose the arrow? (more of a usability observation than accessibility I guess)
- On certain screen sizes, the speaker-info box becomes disconnected from the speaker (approx 640-770px and 935-995px)

Holler if any questions/clarifications needed!

**Comment by jscaffrey** [comment]
Thank you @yarnetsky!! I will work on these. Anything that lingers can turn into its own separate issue.

**Comment by jscaffrey** [comment]
Quick update - I am moving this Thursday and this has impacted my ability to work on this. I might not have a chance to work on these before our meeting, but I appreciate the analysis and understand the issues as you've described them.

**Comment by yarnetsky** [comment]
If there's anything in particular, I can probably tackle a few issues.

**Comment by jscaffrey** [comment]
@yarnetsky created child issue to substitute for this parent issue. Closing this parent issue.
